railgun/internal/ctrlutils/ingress-status.go

```diff
@@ -124,83 +130,109 @@ func toKnativeLBStatus(coreLBStatus []apiv1.LoadBalancerIngress) []knative.LoadB
 	return res
 }
 
-func retrieveNSAndNM(svc file.FService) (string, string, error) {
-	var name, namespace string
-	routeInf := strings.Split(*(svc.Routes[0].Name), ".")
-	namespace = routeInf[0]
-	name = routeInf[1]
-	if len(namespace) == 0 || len(name) == 0 {
-		return "", "", fmt.Errorf("configured route information is not completed which should not have happened")
-	}
-	return namespace, name, nil
-}
-
 // UpdateIngressV1 networking v1 ingress status
 func UpdateIngressV1(ctx context.Context, logger logr.Logger, svc file.FService, cli *clientset.Clientset,
 	ips []string) error {
-	namespace, name, err := retrieveNSAndNM(svc)
-	if err != nil {
-		return fmt.Errorf("failed to generate UDP client. err %v", err)
-	}
-
-	ingCli := cli.NetworkingV1().Ingresses(namespace)
-	cli.NetworkingV1beta1()
-	if err != nil {
-		return fmt.Errorf("failed to generate UDP client. err %v", err)
-	}
-
-	retry := 0
-	for retry < statusUpdateRetry {
-		curIng, err := ingCli.Get(ctx, name, metav1.GetOptions{})
-		if err != nil || curIng == nil {
-			log.Errorf("failed to fetch Ingress %v/%v: %v. retrying...", namespace, name, err)
-			retry++
-			time.Sleep(time.Second)
-			continue
-		}
+	for _, route := range svc.Routes {
+		routeInf := strings.Split(*((*route).Name), ".")
+		namespace := routeInf[0]
+		name := routeInf[1]
+		log.Infof("route name %s namespace %s", name, namespace)
```

Comment:
small a suggestion for an alternative log entry that might help add specificity:
```suggestion
		log.Infof("updating status for v1.Ingress route: name %s namespace %s", name, namespace)
```